cimloader/databases/neo4j.py

```diff
@@ -13,15 +9,14 @@
 from neo4j.exceptions import DriverError, Neo4jError
 
 from cimloader.databases import ConnectionInterface, ConnectionParameters, Parameter, QueryResponse
-# from cimgraph.databases.neo4j import Neo4JConnection
 
 import rdflib
 # from rdflib import Graph, Namespace
 from rdflib.namespace import RDF
 
 _log = logging.getLogger(__name__)
 
-class Neo4jUploader(ConnectionInterface):
+class Neo4jConnection(ConnectionInterface):
```

Comment:
Neo4jConnection needs to be renamed to avoid potential ambiguity with cimgraph.databases.neo4j.Neo4jConnection